docs/src/bootstrap.md

```diff
@@ -594,15 +594,23 @@ spec:
       name: cluster-example-ca
       key: ca.crt
 ```
+
 #### Configure the application database
 
 We also support to configure the application database for cluster which bootstrap
 from a live cluster, just like the case of `initdb` and  `recovery` bootstrap method.
 If the new cluster is created as a replica cluster (with replica mode enabled), application
 database configuration will be skipped.
 
-The following example configure the application database `app` with password in
-supplied secret `app-secret` after bootstrap from a live cluster.
+!!! Important
+    While the `Cluster` is in recovery mode, no changes to the database,
+    including the catalog, are permitted. This restriction includes any role
+    overrides, which are deferred until the `Cluster` transitions to primary.
+    During this phase, users remain as defined in the source cluster.
```

Comment:
I'd be explicit about "this" here. Also, "roles" to be consistent with what we say further down.
```suggestion
    During the recovery phase, roles remain as defined in the source cluster.
```